packages/remix-react/components.tsx

```diff
@@ -666,22 +666,31 @@ export function Meta() {
         // Open Graph tags use the `property` attribute, while other meta tags
         // use `name`. See https://ogp.me/
         let isOpenGraphTag = name.startsWith("og:");
-        return name === "title" ? (
-          <title key="title">{value}</title>
-        ) : ["charset", "charSet"].includes(name) ? (
-          <meta key="charset" charSet={value as string} />
-        ) : Array.isArray(value) ? (
-          value.map((content) =>
+
+        if (["charset", "charSet"].includes(name)) {
+          return <meta key="charset" charSet={value as string} />;
+        }
+
+        if (name === "title") {
+          return <title key="title">{value}</title>;
+        }
+
+        if (Array.isArray(value)) {
```

Comment:
I think the array case should probably get into recursion for this logic rather than duplicating just the `isOpenGraphTag` logic. Maybe we should extract this logic into a function to make that possible.

Reply:
Based on the code I think it was already and I seem to recall this is documented somewhere.

Reply:
I only enabled objects, not array of objects.